tests/test_nsjail.py

```diff
@@ -100,6 +100,27 @@ def test_null_byte_value_error(self):
         self.assertEqual(result.stdout, "ValueError: embedded null byte")
         self.assertEqual(result.stderr, None)
 
+    def test_print_bad_unicode_encode_error(self):
+        result = self.nsjail.python3("print(chr(56550))")
+        self.assertEqual(result.returncode, 1)
+        unicode_traceback = (
+            "Traceback (most recent call last):\n"
+            '  File "<string>", line 1, in <module>\n'
+            "UnicodeEncodeError: 'utf-8' codec can't encode character '\\udce6'"
+            " in position 0: surrogates not allowed\n"
+        )
```

Comment:
I'm not sure if asserting the exact traceback is the right approach to testing this. The way I see it, we don't care how Python deals with it as long as our program doesn't crash trying to get the output. In fact, do we even care if the exit code is 1?

Reply:
I think we do care that it exits with 1, as that indicates the python process has successfully thrown an error message like we've expected. I was trying to find good ways of asserting that it simply returned a UnicodeEncodeError, as that's really what I wanted to check.

Reply:
On second thought, it's useful to assert the exit code to distinguish this case from the case of snekbox catching the error and handling it. Asserting the traceback is needed cause the exit code on its own would be too broad of an assertion. That being said, do you think simplifying it to `assert "UnicodeEncodeError" in output` would be sufficient?

Reply:
I think so. I'll go with that